android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -223,6 +224,10 @@
   protected static final String KMFilename_KmwCss = "kmwosk.css";
   protected static final String KMFilename_Osk_Ttf_Font = "keymanweb-osk.ttf";
   protected static final String KMFilename_Osk_Woff_Font = "keymanweb-osk.woff";
+
+  public static final String KMFilename_InstalledLexicalModelsList = "lexical_models_list.json";
```

Comment:
```suggestion
  public static final String KMFilename_Installed_LexicalModelsList = "lexical_models_list.json";
```
that matches the Keyboard identifier